sdks/python/apache_beam/io/fileio.py

```diff
@@ -270,6 +295,8 @@ def _rename_batch(batch):
       """_rename_batch executes batch rename operations."""
       source_files, destination_files = batch
       exceptions = []
+      if self._file_system is None:
```

Comment:
Do you need to add  `@check_accessible(['file_path_prefix', 'file_name_suffix'])`?